### DISCUSSION TITLE
Switching Server

### DISCUSSION THREAD

**Discussion by JonFP** [codefori/vscode-ibmi#595] [IBM i] [answered]
In the getting started tutorial it talks about having multiple connections. But says nothing about how to switch. Is the only way to switch from one connection to another to disconnect first?  And is there a simpler way of disconnecting than using F1 and selecting the disconnect action?

I see from an earlier answer that VS Code won't support two active connections in one window, so right now my solution is to create a new window so that I have one for each connection. 

Also is there an easy way to get back to "Getting Started" after you close it?  The "Get Started" link in the Help & Support tab takes you to the on-line doc pages not to the built-in stuff.

**Comment by worksofliam** [comment, marked as answer]
> is there a simpler way of disconnecting than using F1 and selecting the disconnect action?

No, but you can map that command (Disconnect from IBM i) to a keybinding and use that instead.

You want to use the Help tab that is part of VS Code.

<img width="921" alt="image" src="https://user-images.githubusercontent.com/3708366/159387340-7e83c13d-84f7-4505-a416-67c713140a40.png">

**Comment by JonFP** [comment]
Thanks - I think I may stick with F1 for now as I'm having enough trouble remembering the differences in keybindings between RDi, ACS, and VS Code!  I'm just getting to old - no grey cells left to remember this stuff!
